client/proposal.go

```diff
@@ -127,6 +127,18 @@ func (r *ProposalResponder) Accept(ctx context.Context, acc ChannelProposalAccep
 	return r.client.handleChannelProposalAcc(ctx, r.peer, r.req, acc)
 }
 
+// SetEgoisticChain sets the egoistic chain flag for a given ledger.
+func (r *ProposalResponder) SetEgoisticChain(egoistic multi.LedgerID) {
```

Comment:
So you can only set the egoistic flag when you receive a proposal? Can't you set this in beforehand when initializing the funders? In the case of Perun-X, the hub would just set this for all chains?

Reply:
Hm okay I see, thanks.

Reply:
Hmm okay, I understand, thanks.